packages/ember-glimmer/lib/environment.js

```diff
@@ -62,6 +62,39 @@ function wrapClassAttribute(args) {
   return args;
 }
 
+function wrapClassBindingAttribute(args) {
+  let hasClassBinding = args.named.has('classBinding');
+
+  if (hasClassBinding) {
+    let { value , type } = args.named.at('classBinding');
+    if (type === 'value') {
+      let [ prop, truthy, falsy ] = value.split(':');
+      let spec;
+      if (prop === '') {
+        spec = ['helper', ['-class'], [truthy], null];
+      } else if (falsy) {
+        let parts = prop.split('.');
+        spec = ['helper', ['-class'], [['get', parts], truthy, falsy], null];
+      } else if (truthy) {
+        let parts = prop.split('.');
+        spec = ['helper', ['-class'], [['get', parts], truthy], null];
+      }
+
+      if (spec) {
+        let syntax;
+        if (args.named.has('class')) {
+          // If class already exists, merge
+          let classValue = args.named.at('class').value;
```

Comment:
This currently won't work with helpers. If we need to support this, we'll need a way to generate a spec from a helper.